internal/testing/alldeps

```diff
@@ -3,31 +3,31 @@ contrib.go.opencensus.io/exporter/aws
 contrib.go.opencensus.io/exporter/ocagent
 contrib.go.opencensus.io/exporter/stackdriver
 contrib.go.opencensus.io/integrations/ocsql
-github.com/aws/aws-sdk-go
 github.com/Azure/azure-amqp-common-go
 github.com/Azure/azure-pipeline-go
 github.com/Azure/azure-service-bus-go
 github.com/Azure/azure-storage-blob-go
 github.com/Azure/go-autorest
+github.com/GoogleCloudPlatform/cloudsql-proxy
+github.com/aws/aws-sdk-go
```

Comment:
It's strange that your listdeps run reordered the file - this could be why Travis is complaining. For example ,if I rerun `sort | uniq` on this file on my Linux machine, `github.com/aws/aws-sdk-go` sorts before the `Azure` modules, but in your PR it's moved after them. 

FWIW I'm using sort v8.28 on bash v4.4.12 running on a x64-64-pc-linux-gnu Debian machine

Reply:
Thanks will take a look.

Reply:
That did not seem to help..

Reply:
https://travis-ci.com/google/go-cloud/jobs/180423337